src/platform.deno.ts

```diff
@@ -0,0 +1,42 @@
+export {
+  Fluent,
+  type FluentBundleOptions,
+  type FluentOptions,
+  type LocaleId,
+  type TranslationContext,
+} from "https://deno.land/x/better_fluent@v0.1.0/mod.ts";
+
+export {
+  FluentType,
+  type FluentValue,
+} from "https://deno.land/x/fluent@v0.0.0/bundle/types.ts";
+export { Scope } from "https://deno.land/x/fluent@v0.0.0/bundle/scope.ts";
+
+export {
+  Context,
+  type Middleware,
+  type NextFunction,
+} from "https://lib.deno.dev/x/grammy@1.x/mod.ts";
+
+export { extname, resolve } from "https://deno.land/std@0.144.0/path/mod.ts";
+
+export function exists(path: string): boolean {
+  try {
+    Deno.lstatSync(path);
+    return true;
+  } catch (err) {
+    if (err instanceof Deno.errors.NotFound) {
+      return false;
+    }
+    throw err;
+  }
+}
+
+export function readLocalesDir(path: string): string[] {
```

Comment:
I chose `readLocalesDir` because I thought it makes more sense. Because, when a user calls `i18n.loadLocalesDir(dir)`, they know that they are loading a directory containing locales. Therefore, I guess `readLocalesDir` (read the locales directory) explains itself more than just getting a directory.

Also, your suggestion in [this review comment](https://github.com/grammyjs/i18n/pull/20#discussion_r902767400) conflicts with the point you made on resuablity of this function.

So, I've come up with a decision, to not to change this function name...

> because I believe, there won't be a case where we're gonna have to read files from a directory again in this plugin -- if there is, we can change.

...but to take your suggestions from [this comment](https://github.com/grammyjs/i18n/pull/20#discussion_r902767400), and make this function return only files ending with `.ftl` format. So, the name of the function have more meaning of what it is doing.

Do you agree with this?

---

> I also would like to avoid read because read is more like the term of C language when we still have to pass the pointer around... (In node and deno we still use this term, but I don't think it was this case)

Deno namespace has a function called `readDir` which does the same. So, I don't think `read` was a bad choice :)

And I don't think we should spend more time on just renaming stuff that is not really important (By important I mean, it is not exported externally. It is just an inside function).